test/consistency-and-style/test_summary_text.py

```diff
@@ -194,14 +177,12 @@ def test_summary_text_available(module_name):
     module = modules[module_name]
     vars = dir(module)
     for name in vars:
-        var = getattr(module, name)
+        var = contributing_builtin_var(module, name)
+        if var is None:
```

Comment:
Good!  Here, we use `var is None`  whereas before `is not None` was avoided in favor of a pep exception